modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * virt//support/monitoring/virt-running-cluster-checkups.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-building-vm-containerdisk-image_{context}"]                                  
+= Building a container disk image for RHEL virtual machines
+
+You can build a custom RHEL 8 OS image in `qcow2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map. 
+
+Run the following procedure in the image builder virtual machine (VM) terminal to build a container disk image for RHEL 8.7 VMs with 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory. The _image builder VM_ is a RHEL 8 VM that can be used to build custom RHEL images.
+
+
+.Prerequisites
+* Install the image builder tool and its CLI (`composer-cli`).
+* Verify that you can build a RHEL 8.7 image:
++
+[source,terminal]
+----
+# composer-cli distros list
+----
++
+[NOTE]
+====
+To run the `composer-cli` commands as non-root, add your user to the `weldr` or `root` groups:
+
+[source,terminal]
+----
+# usermod -a -G weldr user
+----
+[source,terminal]
+----
+$ newgrp weldr
+----
+====
+* Install the `virt-customize` tool by using the following command:
++
+[source,terminal]
+----
+# dnf install libguestfs-tools
+----
+* Install `podman`.
+
+.Procedure
+. Create an image blueprint file in TOML format which contains the packages to be installed, kernel customizations, and the services to be disabled during boot time:
++
+[source,terminal]
+----
+$ cat << EOF > dpdk-vm.toml
+name = "dpdk_image"
+description = "Image to use with the DPDK checkup"
+version = "0.0.1"
+distro = "rhel-87"
+
+[[packages]]
+name = "dpdk"
+
+[[packages]]
+name = "dpdk-tools"
+
+[[packages]]
+name = "driverctl"
+
+[[packages]]
+name = "tuned-profiles-cpu-partitioning"
+
+[customizations.kernel]
+append = "default_hugepagesz=1GB hugepagesz=1G hugepages=8 isolcpus=2-7"
+
+[customizations.services]
+disabled = ["NetworkManager-wait-online", "sshd"]
+EOF
+----
+
+. Push the blueprint file to the image builder tool by using its CLI:
++
+[source,terminal]
+----
+# composer-cli blueprints push dpdk-vm.toml
+----
+
+. Generate the system image by specifying the blueprint name and output file format. The Universally Unique Identifier (UUID) of the image is displayed when you start the compose process.
```

Comment:
```suggestion
. Enter the following command to generate the system image by specifying the blueprint name and output file format. The Universally Unique Identifier (UUID) of the image is displayed when you start the compose process.
```

If you're going to mention that the UUID of the image is displayed, you might want to provide an ".Example output". Just a thought.

Reply:
@orelmisan Do you think it would be helpful to provide an example here to show the UUID of an image?

Reply:
Its just a long number of letters and digits.
I don't think it matters.